src/libraries/System.Data.Common/src/System/Data/Common/DbException.cs

```diff
@@ -21,5 +21,6 @@ public abstract class DbException : System.Runtime.InteropServices.ExternalExcep
         protected DbException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) : base(info, context)
         {
         }
+        public virtual string? SqlState => null;
```

Comment:
nit: please introduce empty line above

Reply:
@Dotnet-GitSync-Bot mentioned triple slash comments in <https://github.com/dotnet/runtime/pull/38881#issuecomment-655019315>. Are they mandatory here?